crates/bevy_math/src/curve/easing.rs

```diff
@@ -903,4 +985,71 @@ mod tests {
             );
         });
     }
+
+    #[test]
+    fn step_config_start() {
+        let config = JumpAt::Start;
+        let num_steps = 4;
+
+        [
+            (0.0, 0.25),
+            (0.25, 0.5),
+            (0.5, 0.75),
+            (0.75, 1.0),
+            (1.0, 1.0),
```

Comment:
Taking a leaf from #17777, could this include a few edge case checks?

```suggestion
            (0.0, 0.25),
            (0.24, 0.25),
            (0.25, 0.5),
            (0.49, 0.5),
            (0.5, 0.75),
            (0.74, 0.75),
            (0.75, 1.0),
            (1.0, 1.0),
```